mixer/pkg/runtime/config/ephemeral.go

```diff
@@ -205,13 +205,15 @@ func (e *Ephemeral) processAttributeManifests(ctx context.Context, errs *multier
 	// using the $out.<field Name> convention, where $out refers to the output object from the attribute generating adapter.
 	// The list of valid names for a given Template is available in the Template.Info.AttributeManifests object.
 	for _, info := range e.templates {
-		log.Debugf("Processing attributes from template: '%s'", info.Name)
-
-		for _, v := range info.AttributeManifests {
-			for an, at := range v.Attributes {
-				attrs[an] = at
-				stats.Record(ctx, monitoring.AttributesTotal.M(1))
-				log.Debugf("Attribute '%s': '%s'", an, at.ValueType)
+		if info.Variety == v1beta1.TEMPLATE_VARIETY_ATTRIBUTE_GENERATOR {
```

Comment:
note: I want to filter out check adapter attribute manifests, since those need to be prefixed by the action name.